js/main.js

```diff
@@ -91,30 +91,26 @@ function buyNow(event) {
   prices.addEventListener('load', function () {
     for (var i = 0; i < this.response.deals.length; i++) {
       storeListings.className = "search-results";
-      var resultHeader = gameTitle;
-      console.log(resultHeader)
       var priceResult = document.createElement('li');
       priceResult.className = "result-row"
       storeListings.appendChild(priceResult);
       var storeId = this.response.deals[i].storeID;
       priceResult.setAttribute("storeid", storeId);
       var storeIcon = document.createElement('img');
       var iconImage;
-      for(var k = 0; k < storesList.length; k++) {
-        if (storeId === storesList[k].storeID) {
-          iconImage = "https://www.cheapshark.com" + storesList[k].images['banner'];
+      for(var j = 0; j < storesList.length; j++) {
+        if (storeId === storesList[j].storeID) {
+          iconImage = "https://www.cheapshark.com" + storesList[j].images['banner'];
+        }
+        if (storeId === storesList[j].storeID) {
+          storeActualName = storesList[j].storeName;
```

Comment:
You don't need two conditionals here, you can do the work for both of these conditionals in one conditional